Code/Tools/SceneAPI/SceneUI/RowWidgets/ManifestVectorWidget.cpp

```diff
@@ -271,8 +272,10 @@ namespace AZ
                                     return object.get() == cast;
                                 }) != m_manifestVector.end(), "ManifestVectorWidget detected an update of a field it doesn't own.");
                             EmitObjectChanged(cast);
+                            return;
```

Comment:
is the return a good early out? Is one EmitObjectChanged() all that is needed?

Reply:
yeah, it's just verifying that it cares about the object change. This matches the logic in ManifestWidgetPage